solr/solr-ref-guide/src/performance-statistics-reference.adoc

```diff
@@ -191,9 +187,7 @@ You can get the statistics shown in the table below with an API request such as
 
 The following statistics are available for each of the caches mentioned above:
 
-// TODO: Change column width to %autowidth.spread when https://github.com/asciidoctor/asciidoctor-pdf/issues/599 is fixed
-
-[cols="25,75",options="header"]
+[%autowidth.spread,width="100%",options="header"]
```

Comment:
This one gets parameter wrapped-up in weird way. I wish there was a way to mark column no-wrap specifically for this one. But it is ok otherwise.

Reply:
There was another example where I decided the autowidth was worse than the manual spaced one, so I removed the TODO but left it as a`[cols=]` style.   If you think it's better to keep the `[cols=]` let me know and I'll revert that change!

Reply:
Should we just keep the old format?